src/update-roster-new.R

```diff
@@ -0,0 +1,363 @@
+scrape_old_roster <- function(team, )
+
+build_rosters <-
+  function(season = nflreadr:::most_recent_season(roster = TRUE)) {
+    # if NGS rosters are not available, default to sleeper API
+    if (season >= 2016) {
+      teams <- ngsscrapR::scrape_teams(season) |>
+        dplyr::filter(!is.na(division_id))
+      # weeks <- purrr::map_dfr(c("REG","POST"), ~ngsscrapR::scrape_schedule(season, .x)) |>
+      #   dplyr::group_by(week, game_type) |>
+      #   dplyr::summarise()
+      # team_week_pair <- teams |>
+      #   dplyr::select(team_id) |>
+      #   dplyr::mutate(join = 1) |>
+      #   dplyr::left_join(weeks |>
+      #                      dplyr::mutate(join = 1),by=c("join")) |>
+      #   dplyr::select(-c(join)) |>
+      #   dplyr::rename(teamId = team_id,
+      #                 seasonType = game_type) |>
+      #   dplyr::mutate(season = !!season)
+
+      cli::cli_alert_info("Obtaining roster data...")
+      rosters <-
```

Comment:
naming this "gsis_rosters" instead of just "rosters" would help make this clearer for future readers